terraform/modules/coreos_asg/main.tf

```diff
@@ -0,0 +1,223 @@
+data "aws_region" "selected" {
+    current = true
+}
+
+data "aws_caller_identity" "current" {}
+
+resource "aws_ecs_cluster" "cluster" {
+    name = "${var.name}"
+}
+
+data "aws_ami" "coreos_stable" {
```

Comment:
Nope, not doubling down on the CoreOS mistake.
We had this discussion already...